testsuite/tests/singlecluster/authorino/caching/metadata/test_caching.py

```diff
@@ -22,12 +22,15 @@ def test_cached(client, auth, module_label, mockserver):
         - both requests return the same result
         - only single external value evaluation occurs. The second response contains cached (in-memory) value
     """
-    response = client.get("/get", auth=auth)
-    data = extract_response(response)[module_label]["uuid"] % None
-    response = client.get("/get", auth=auth)
-    cached_data = extract_response(response)[module_label]["uuid"] % None
+    response1 = client.get("/get", auth=auth)
```

Comment:
why this usage of two variables (`response1` and `response2`) instead of using just `response` variable here, is that due to easier debugging?

Reply:
I changed it to make it similar to the other test, not cached. But overall, I try to avoid reassigning variables in Python. I think it can cause some unintended bugs